akvo/templates/myrsr/project_editor/project_editor.html

```diff
@@ -29,7 +29,13 @@ <h3>{% trans 'Project Editor' %} <span class="glyphicon glyphicon-info-sign info
                 <span>{% trans 'Date created' %}:</span>
                 <time datetime="YYYY-MM-DDThh:mm:ssTZD">{{ project.created_at }}</time><br/>
                 <span>{% trans 'Last modified' %}:</span>
-                <time datetime="YYYY-MM-DDThh:mm:ssTZD">{{ project.last_modified_at }}</time>
+                <time datetime="YYYY-MM-DDThh:mm:ssTZD">{{ project.last_modified_at }}</time><br/>
+                {% with last_user=project.last_modified_by %}
+                    {% if last_user %}
+                        <span>{% trans 'Last modified by' %}:</span>
+                        <time>{{ user.first_name }}  {{ user.last_name }}</time>
```

Comment:
You can use User.get_full_name() here. I fixed that method in #3217 so that it displays the user's ID if the name is empty.

Reply:
Fixed, thanks!